config/webpack.config.js

```diff
@@ -7,6 +7,7 @@ var ForemanVendorPlugin = require('@theforeman/vendor').WebpackForemanVendorPlug
 var UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 var StatsWriterPlugin = require("webpack-stats-plugin").StatsWriterPlugin;
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin');
```

Comment:
This reduces the plugin css files size dramatically by removing duplications 
before:
![Screenshot from 2020-11-03 14-20-07](https://user-images.githubusercontent.com/11807069/97993136-51651b00-1dec-11eb-837f-72e05d03eb14.png)

after:
 ![Screenshot from 2020-11-03 14-18-22](https://user-images.githubusercontent.com/11807069/97993092-427e6880-1dec-11eb-845e-91cf92dfe0dd.png)

Reply:
It's cool ! although I've got doubts we need all the css bundled in, that would deserve an investigation.
Can we split the CSS optimization to a different PR? it will need a packaging PR, so we would be blocked on that and I feel it's a separate issue :)

Reply:
Sure, makes sense :+1: